Snakefile

```diff
@@ -0,0 +1,267 @@
+import pandas as pd
+
+metadata = pd.read_csv("inputs/metadata.csv")
+SAMPLES = metadata['run_accession'].unique().tolist()
+KSIZES = [21, 31, 51]
+LINEAGES=['bacteria', 'viral', 'archaea', 'fungi', 'protozoa']
+
+rule all:
+    input: expand("outputs/sourmash_taxonomy/{samples}ass-vs-genbank-2022.03-k{ksize}.with-lineages.csv", ksize = KSIZES, samples = SAMPLES)
+
+rule k21:
+    input: expand("outputs/sourmash_taxonomy/{samples}ass-vs-genbank-2022.03-k{ksize}.with-lineages.csv", ksize = 21, samples = SAMPLES)
+
+rule k31:
+    input: 
+        expand("outputs/sourmash_taxonomy/{samples}ass-vs-genbank-2022.03-k{ksize}.with-lineages.csv", ksize = 31, samples = SAMPLES),
+        expand("outputs/sourmash_compare/comp_k{ksize}.csv", ksize = 31),
+        expand("outputs/sourmash_sketch_csv/{samples}_k{ksize}.csv", ksize = 31, samples = SAMPLES)
+
+rule k51:
+    input: expand("outputs/sourmash_taxonomy/{samples}ass-vs-genbank-2022.03-k{ksize}.with-lineages.csv", ksize = 51, samples = SAMPLES)
+
+##########################################################
+## Download sourmash databases & taxonomy files
+##########################################################
+
+rule download_genbank_bacteria_zip_k21:
```

Comment:
I'm assuming this will change to pulling from S3 once we have the databases bucket figured out

Reply:
yes! or be an ifelse/parameterized statement to avoid download. Although...download ended up being less horrible than I thought...I was using some weird links originally and it was taking FOREVER to get the data, but then I updated to these and it was 50mb/s and that was perfectly acceptable. See https://github.com/sourmash-bio/sourmash/issues/2179 and https://github.com/sourmash-bio/sourmash/issues/2136